gradle/verification-metadata.xml

```diff
@@ -768,6 +772,11 @@
             <pgp value="47eb6836245d2d40e89dfb4136d4e9618f3adab5"/>
          </artifact>
       </component>
+      <component group="io.grpc" name="grpc-context" version="1.19.0">
+         <artifact name="grpc-context-1.19.0.jar">
+            <pgp value="b02335aa54ccf21e52bbf9abd9c565aa72ba2fdd"/>
```

Comment:
Since the key has been added as a trusted key for these coordinates, this entry can be removed.